scripts/ayp.coffee

```diff
@@ -254,7 +254,8 @@ module.exports = (robot) ->
           ]
           Twitter.mediaTweet strip.info.image_jpeg, "#{prefix} - #{strip.info.url}", (err, tweet, url) =>
             sorry = msg.random ["sorry", "it's your fault"]
-            return msg.reply "Well, I fucked tweeting that RIGHT up, #{sorry}: #{err}" if err
+            # We will fix this when we fix twitter
+            return #msg.reply "Well, I fucked tweeting that RIGHT up, #{sorry}: #{err}" if err
```

Comment:
That should still ideally be `return if err` so that the happy path is still *possible*

Reply:
If only I could read 🤦